### ISSUE TITLE
Result value not visible.

### ISSUE THREAD

**Issue by RohDonghyeon** [MoonBlvd/Detection-of-Traffic-Anomaly#12]
DoTA dataset - Download  complete.
Annotation for VAD - Ok

Both of the above course have been completed and the final process has been carried out.
Extracted data for FOL model training and testing.

However, no results are saved and visible after the operation as shown in the figure below.
![No_result](https://user-images.githubusercontent.com/33896710/97951170-a6009a00-1ddc-11eb-99e6-f756b53766d2.png)

**Comment by MoonBlvd** [comment]
Hi @RohDonghyeon , this repo is suppose to be the dataset repo not the repo for the methodology we used in the paper. "sort_output" and "flownet2" are directories where we save our pre-computed object tracking and optical flow output. We have not provided these two in the repo since they suppose to be together with the methodology instead of the dataset. We  do not want users to feel that thay have to use "sort" or "flownet2" if they use our dataset. However, we provide links to download these preprocessed features we used to train our method in the README file, you can download them if you want to take a look.

**Comment by RohDonghyeon** [comment]
In other words, it's not a methodology for learning and testing, it's a report on the provision of datasets, right?
Then, do you have any plans to provide a code to conduct the train, test, and view the results?

Also I couldn't find
![image](https://user-images.githubusercontent.com/33896710/98052434-94210480-1e79-11eb-908e-b11c8b40bc4a.png)
![image](https://user-images.githubusercontent.com/33896710/98052442-997e4f00-1e79-11eb-9dc3-f2199dfdb97a.png)
Not found in readme file.

**Comment by MoonBlvd** [comment]
Correct. We are still working on cleaning the code and will release is soon. As to the download link for the pre-extracted features, please find it in the README file.

**Comment by RohDonghyeon** [comment]
Thank Thank you for your sincere reply.
I'll see you when the code comes out!

**Comment by sumitmishra209** [comment]
Hi @MoonBlvd  I am trying to load and see the dataset but it is returning nothing the len is also zero. I have checked that directory are set well. Also I have noticed that in dataset initialization img_dir is never used which has all the frames. Can you please see the issues. Also, if possible can you please update or send me the working codes of this repo along with models that you experiment with in DoTA paper. My email id is sumitmishra209@gmail.com 

Thanks for your help :)  

![image](https://user-images.githubusercontent.com/36421656/147257220-7d817f35-8107-48cf-b77c-6b90b004c596.png)

![image](https://user-images.githubusercontent.com/36421656/147256917-899e0deb-de73-42f8-a67f-27e705f54a42.png)

**Comment by ghadeerAbdulmajeed** [comment]
Hi @sumitmishra209  did you get the code ?
